scripts/gulpfiles/release_notes.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Gulp scripts for creating release notes.
+ */
+
+var gulp = require('gulp');
+var fetch = require('node-fetch');
+
+const baseUrl = "https://api.github.com/graphql";
+
+async function generateReleaseNotes(done) {
+  const goalDate = Date.parse('2021-03-01');
+  const githubToken = process.argv[4];
+  const headers = {
+    "Content-Type": "application/json",
+    Authorization: "bearer " + githubToken,
+  };
+
+
+  var note = `{Insert greeting},
+
+The {Insert Month and Year} release is out! In this release:
+
+{List major features in both core and samples}
+
+🎉 **Kudos** 🎉
+{Thank forums contributors and issue filers}
+
+**Third Party Plugins**
+{List third party plugins that have been released in the last quarter}
+
+**Core contributions**
+{Reorganize into thank-yous and check for missed people}
+${await getContributors('blockly', 'develop', goalDate, headers)}
+
+**Samples contributions**
+{Reorganize into thank-yous and check for missed people}
+${await getContributors('blockly-samples', 'master', goalDate, headers)}
+
+🔨 **Breaking Changes in core** 🔨 
+${await getBreakingChanges(headers)}
+
+Full release notes are below. As always, we welcome bug reports and pull requests!
+
+Cheers,
+{Your Name}
+
+**Issues Closed**:
+${await getClosedIssues(headers)}
+
+**Pull Requests**:
+${await getMergedPrs(headers)}`;
```

Comment:
Newline before the closing backtick, both so output ends with newline and to make it easier to spot the closing delimiter of the long multi-line template literal.